docs/topics/writing_templates.rst

```diff
@@ -111,7 +111,7 @@ See :ref:`image_tag` for full documentation.
 Rich text (filter)
 ~~~~~~~~~~~~~~~~~~
 
-This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images, and links made in the Wagtail editor, into fully-baked HTML ready for display.
+This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images and links made in the Wagtail editor, into fully-baked HTML ready for display.
```

Comment:
I think this works better with the extra comma. "expands ... into" is a single expression - with two commas, this can be read as one expression that's interrupted by a parenthetical comment:

> Importantly, it also expands internal shorthand references to embedded images (and links made in the Wagtail editor) into fully-baked HTML ready for display.

but with only one comma, it's incorrectly splitting the phrase into two. We could remove both, but then it's a bit too long to read without breaks.

Reply:
sorry, I wasn't clear there - the parentheses weren't a suggestion, they were an explanation of how the existing sentence should be read. I think this should be kept as the original version, with two commas.

Reply:
My bad for the misinterpretation, but wait this is getting confusing.

**This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images and links made in the Wagtail editor into, fully-baked HTML ready for display.**

Should the final edit look like above ??

Reply:
No, I think this should be kept as the original text:

> This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images, and links made in the Wagtail editor, into fully-baked HTML ready for display.

Reply:
Should be ok now .
have a look.